specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -654,6 +669,20 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "deleteType",
+            "description": "Optional. Specifies that the delete operation should be purge for soft delete volume group. This cannot be used with active volume group delete. The value of deleteType can only be 'permanent'.",
```

Comment:
This description might not be so clear to me. 

"the delete operation should be purge", it's the VG be purge, or the delete operation be purge?
How about not use "purge", but use permanent deleted to make terminology aligned?